From 63fe9d73460116e5aba965f080ad7821cfb2826c Mon Sep 17 00:00:00 2001
From: Hatem ElKharashy <hatem.elkharashy@qt.io>
Date: Fri, 8 Sep 2023 13:31:41 +0300
Subject: [PATCH 44/81] Add Radial Gradient default value

Adds a default value for radial gradient's radius
attribute to prevent error in parsing when the
attribute is missing from the elment

Change-Id: Ic9f0f8d7a76aa1f5033aae0e5685f60df8816fbf
Reviewed-by: Eskil Abrahamsen Blomfeldt <eskil.abrahamsen-blomfeldt@qt.io>
---
 src/svg/qsvghandler.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/svg/qsvghandler.cpp b/src/svg/qsvghandler.cpp
index 36b9b0f..cdc78d3 100644
--- a/src/svg/qsvghandler.cpp
+++ b/src/svg/qsvghandler.cpp
@@ -3015,7 +3015,7 @@ static QSvgStyleProperty *createRadialGradientNode(QSvgNode *node,
     if (!cy.isEmpty())
         ncy = toDouble(cy);
 
-    qreal nr = 0.0;
+    qreal nr = 0.5;
     if (!r.isEmpty())
         nr = toDouble(r);
     if (nr <= 0.0)
-- 
2.42.1

